### DISCUSSION TITLE
Provision Http Extension

### DISCUSSION THREAD

**Discussion by jannotti-glaucio** [eclipse-edc/Connector#4608] [Q&A]
Guys, anyone here already used the provision http extension (https://github.com/eclipse-edc/Connector/tree/main/extensions/control-plane/provision/provision-http) ? I'm trying to use it, but the payload example on the swagger documentation is not working:
https://eclipse-edc.github.io/Connector/openapi/management-api/#/Contract%20Negotiation%20V3/initiateContractNegotiationV3

**Comment by ndr-brt** [comment]
that extension didn't get a lot of love ultimately, you could take a look at the E2E test that's covering that feature:
https://github.com/eclipse-edc/Connector/blob/c5dbf9dad5fd474e95a777ace5cff22cc3a9c216/system-tests/e2e-transfer-test/runner/src/test/java/org/eclipse/edc/test/e2e/TransferPullEndToEndTest.java#L208

for the non-working documentation, could you open an issue about it? thanks

**Comment by jannotti-glaucio** [comment]
Thanks @ndr-brt for the help, but I already understood how the extension works. My issue is related to knowing how to call the provision endpoint to move the transfer process from the provision requested status to the next status on the process.

The latest swagger is defining the /provision payload as below, but this format doesn`t work and there is no information about what we need to inform for each tag:
![image](https://github.com/user-attachments/assets/1288906a-a27f-42e3-99d5-88878d1f6945)

I will open the documentation issue, but I need to know where I can find any example or documentation explaining how to use this endpoint to make the transfer process continue. It can be a source code comment or a test case.

**Reply by ndr-brt** [reply]
the test I linked in the previous message does exactly this, please look at the `HttpProvisionerCallback` object:
https://github.com/eclipse-edc/Connector/blob/c5dbf9dad5fd474e95a777ace5cff22cc3a9c216/system-tests/e2e-transfer-test/runner/src/test/java/org/eclipse/edc/test/e2e/TransferPullEndToEndTest.java#L302-L325

plus, "doesn't work" is not a good report: some errors or stack traces would be really helpful
